src/components/List/RowContent.js

```diff
@@ -347,7 +398,30 @@ const ListRowContent = ({
               {isOpen && onExpandShow === "title" ? title : subTitle}
             </MultilineText>
           </ContentColumn>
-        </Row>
+
+          <WithOptions variant={variant}>
+            <OptionsContent>
+              <MobileLink
+                linkUrl={linkUrl}
+                data-index={index}
+                onClick={onOverflowClick}
+                className="link--variant-with-link"
+              >
+                {linkTitle}
+              </MobileLink>
+              <DesktopLink
+                href={linkUrl}
+                data-index={index}
+                className={classnames({
+                  "link--variant-with-link": !linkUrl.length,
```

Comment:
This `className` seems repetitive... maybe change it to something like `link--option`?

Reply:
Rename to `link--row-with-options`.

Reply:
Perfect!